### DISCUSSION TITLE
GitHub tickets

### DISCUSSION THREAD

**Discussion by haikaru17** [community/community#138395] [Enterprise]
### Select Topic Area

Question

### Body

I contacted GitHub support over 2 months ago and still haven't heard back. I have asked several times if there is any news etc. about my case but never received an answer.
Since this is a problem that only GitHub can help me with, I have to rely on their support.

Is it normal that this takes so long?
I have never had contact with the support of GitHub before and therefore have no experience with it.

**Comment by itisrude** [comment]
Hey haikaru17, 



I'm sorry to hear about your experience with GitHub support. While GitHub's response times can sometimes vary depending on the complexity of the issue and the volume of tickets they receive, two months without any reply is definitely a longer-than-usual wait. 



A couple of things you could try: 



1. **Check Spam/Junk Mail**: It's possible their response might have ended up in your spam folder, so give that a quick check just in case. 



2. **Follow Up Again**: If you haven't already, try following up directly on the support thread. Sometimes messages can get missed or overlooked. 



3. **Raise the Issue via Social Channels**: Some users have found success reaching out to GitHub via their [Twitter account](https://twitter.com/github) or community forums when their support tickets have been delayed. 



If your issue is something critical to your workflow, mentioning that in the follow-up may help speed things up. 



Hopefully, you hear back soon!

**Comment by davevad93** [comment]
Hi @haikaru17 ,

GitHub may take a while to reply to a ticket, depending on the ticket category and account plan, for standard users the waiting time is usually longer and yes,  there have been cases where they have taken more than 2-3 months. Also keep in mind that each day they get a lot of requests, so there are many pending tickets that they have to answer.

**Comment by baharbaba** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Comment by Hruthik0x** [comment]
Its been over 2 weeks, I did not receive any response from github support. @github-staff please respond to my ticket.
